test/parallel/test-http-request-dont-override-options.js

```diff
@@ -4,14 +4,15 @@ const common = require('../common');
 const assert = require('assert');
 const http = require('http');
 
-let requests = 0;
 
-http.createServer(function(req, res) {
+const server = http.createServer(common.mustCall(function(req, res) {
   res.writeHead(200);
   res.end('ok');
+}));
 
-  requests++;
-}).listen(0, function() {
+server.listen(0, nextTest);
+
+function nextTest() {
```

Comment:
Can this be just inlined above? The name isn't very descriptive (and could confuse someone in the future) since there's only 1 test, not many.

`server.listen(0, () => {...});`

Reply:
@apapirovski : Thanks for the feedback. I've updated the PR as per the comment and also shortened the commit message. Kindly review the PR now. Thanks !